src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/SegmentValidations/SegmentValidationStatus.cs

```diff
@@ -0,0 +1,9 @@
+namespace StateRecorder.BotSegments.Models.SegmentValidations
+{
+    public enum SegmentValidationStatus
+    {
+        PASSED,
+        FAILED,
+        UNKNOWN
```

Comment:
Would you mind outlining in what cases it could be UNKNOWN? I am making assumptions but I would rather be clear.

Reply:
Comments added 👍